### DISCUSSION TITLE
告警经常不触发，帮忙看一下

### DISCUSSION THREAD

**Discussion by hedongfang** [clickvisual/clickvisual#929] [Q&A] [answered]
告警配置：
<img width="788" alt="image" src="https://user-images.githubusercontent.com/49864372/218385188-d7dbbda2-d060-4c1e-b6f3-5bd436fd564c.png">

<img width="787" alt="image" src="https://user-images.githubusercontent.com/49864372/218385358-795fdfe3-7032-458b-8ae6-71bcc70a4824.png">

graphite_rollup配置
![image](https://user-images.githubusercontent.com/49864372/218385294-bbd85692-30b7-4e18-99d6-62f7e456b46e.png)

现在调整了触发条件和graphite_rollup
![image](https://user-images.githubusercontent.com/49864372/218385807-b5bb79dd-d997-4753-b220-214f205bc918.png)
![image](https://user-images.githubusercontent.com/49864372/218385852-f4f69f70-de73-49d1-94a2-2eb57d131e40.png)

**Comment by kl7sn** [comment, marked as answer]
https://clickvisual.gocn.vip/zh/clickvisual/03funcintro/alarm-function-configuration-description.html#%E6%9E%B6%E6%9E%84%E8%AF%B4%E6%98%8E

告警数据的设计是通过物化视图降数据写入 metrics.samples 表中，可以先观测下该表中的数据是否符合预期。

graphite_rollup 的配置调整，影响 metrics.samples 表数据聚合
- 调整前：每秒的数据取和
- 调整后：每五秒的数据取平均

触发条件：这个和 prometheus 的 rules 有关系，应该不在排查范围内，我们确保 metrics.samples 的数据是准确的应该就可以了。
